tests/test_scylla_repository.py

```diff
@@ -8,9 +8,20 @@
 @pytest.mark.repo_tests
 @pytest.mark.skip("slow integration test")
 class TestScyllaRepository:
-    def test_setup_release_oss(self):
-        cdir, version = scylla_setup(version="release:4.2.1", verbose=True)
-        assert version == '4.2.1'
+    @pytest.mark.parametrize(argnames=['version', 'expected_version_string'], argvalues=[
+        ("release:5.1", '5.1'),
+        ("release:5.0", '5.0'),
+        ("release:4.6", '4.6'),
+        ("release:4.5", '4.5'),
+        ("release:4.4", '4.4'),
+        ("release:4.3", '4.3'),
+        ("release:4.2", '4.2'),
+        ("release:4.1", '4.1'),
+        ("release:4.0", '4.0'),
```

Comment:
Any reason to keep those EOS releases?

Reply:
since there's still dtest test code that refers to them.

Reply:
Need to open an issue to clean up that dtest code?

Reply:
https://github.com/scylladb/scylla-dtest/pull/3065